java-client/build.gradle.kts

```
@@ -184,12 +184,12 @@ dependencies {
 
     // EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
     // https://github.com/eclipse-ee4j/jsonp
-    api("jakarta.json:jakarta.json-api:2.0.1")
+    api("jakarta.json:jakarta.json-api:2.1.0")
```

Let's keep version 2.0.1 for now, and update to 2.1.0 separately. This will allow backporting this PR down to version 7.17.

```suggestion
    api("jakarta.json:jakarta.json-api:2.0.1")
```